charts/certificates/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: kcp-certificates
+description: A prototype of a multi-tenant Kubernetes control plane for workloads on many clusters
+
+# version information
+version: 0.0.2
+appVersion: "0.21.0"
+
+# optional metadata
+type: application
+home: https://www.kcp.io/
+keywords:
+  - kcp
+  - kubernetes
+  - multitenancy
+  - controlplane
+sources:
+  - https://github.com/kcp-dev/helm-charts
+  - https://github.com/kcp-dev/kcp
+icon: https://raw.githubusercontent.com/kcp-dev/kcp/main/contrib/logo/blue-green.svg
```

Comment:
What's the idea behind moving certificates into a dedicated chart? Can't / Shouldn't they be deployed as part of the respective component (e.g. cache certificates in the `cache` Chart). Would it maybe make sense to just create the PKI in this chart?

Reply:
Also, name should be `certificates`.

Reply:
The first deployment should be certificate agnostic. And when deploying shards ideally they will be globally distributed. Certificates in the real world is pre-requisite, distributed before. 
helm chart & cert-manager is just one way of doing things.
This decouples them even more.

Reply:
and when doing phased deployments they need need to be already present :/